### ISSUE TITLE
deno: Report/record bids validator + hed validator in the bids-validator output

### ISSUE THREAD

**Issue by yarikoptic** [bids-standard/bids-validator#2178]
ATM none seems reported in deno report output. unclear if using most recent hed validator

**Comment by effigies** [comment]
Agreed. It includes the schema version in the JSON output, but we should be putting the validator version in, and then reporting that.
